### DISCUSSION TITLE
How to create default order on Table ?

### DISCUSSION THREAD

**Discussion by rahmanramsi** [filamentphp/filament#2152] [Help]
I want to make a column as a default order. How to do it on filament table ?

**Comment by Z3d0X** [comment]
https://filamentphp.com/docs/2.x/admin/resources#sorting-a-column-by-default

**Reply by rahmanramsi** [reply]
Thanks for the answer, I'm not using admin package.

Already found it.
```php
protected function getDefaultTableSortColumn(): ?string
{
  return 'name';
}
```

I think there's no documentation about this on Tables.

**Reply by khanakia** [reply]
@rahmanramsi which file should add this function ?

**Reply by danharrin** [reply]
For the admin panel: https://filamentphp.com/docs/2.x/admin/resources/listing-records#sorting-a-column-by-default
